pkg/registry/storage/volumeattachment/strategy.go

```diff
@@ -47,6 +48,22 @@ func (volumeAttachmentStrategy) NamespaceScoped() bool {
 	return false
 }
 
+// GetResetFields returns the set of fields that get reset by the strategy
+// and should not be modified by the user.
+func (volumeAttachmentStrategy) GetResetFields() map[fieldpath.APIVersion]*fieldpath.Set {
+	fields := map[fieldpath.APIVersion]*fieldpath.Set{
+		"storage.k8s.io/v1": fieldpath.NewSet(
```

Comment:
note: v1beta1 is missing here on purpose as status is allowed for modification there (see line 78)